src/go/privileged-service/pkg/manage/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright © 2024 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package manage is used to install, uninstall, start, and stop the privileged
+// service.
+package manage
+
+func init() {
+}
```

Comment:
Why is the `init` function needed?

It exists in a couple other `doc.go` files too, but not all of them.

Reply:
That's just a dummy function that's never used, but exists so that there's always some code in the package (after filtering by build tag).  Otherwise we can end up with errors about having nothing in the package.

I'm using `init()` because that specific function can exist multiple times in a package (or even file) with no ill effects; see [documentation](https://go.dev/ref/spec#Package_initialization) (scroll down a little to `func init() { … }`).

Reply:
Ok. I was just confused because e.g. `src/go/privileged-service/cmd/doc.go` doesn't have an `init` function.